stackslib/src/chainstate/stacks/auth.rs

```diff
@@ -1407,12 +1407,10 @@ impl TransactionAuth {
                 };
                 origin_supported && sponsor_supported
             }
-            TransactionAuth::Standard(ref origin) => match origin {
-                TransactionSpendingCondition::OrderIndependentMultisig(..) => {
-                    epoch_id >= StacksEpochId::Epoch30
-                }
-                _ => true,
-            },
+            TransactionAuth::Standard(TransactionSpendingCondition::OrderIndependentMultisig(
+                ..,
+            )) => epoch_id >= StacksEpochId::Epoch30,
+            _ => true,
```

Comment:
I don't like this function, it should never use `_ => true` (it should explicitly handle all cases), because if we ever add to these enums it will still build and return incorrect values

You don't need to re-write the entire function, but I think this Clippy suggestion makes it even worse